src/index.js

```diff
@@ -4,5 +4,5 @@ import App from './components/App';
 
 ReactDOM.render(
     <App />,
-    document.getElementById('app')
+    document.getElementById('app'),
```

Comment:
i'd have to dig through the code, but i'm pretty sure that the repo follows the "use commas at end" convention for things like this, i'd prefer if this non-functional comma change were not included